src/intTest/java/uk/nhs/digital/nhsconnect/nhais/outbound/controller/FhirControllerIntegrationTest.java

```diff
@@ -119,5 +122,12 @@ void whenRemovalInvalidJsonStructure_thenReturns400() throws Exception {
             .andExpect(status().isBadRequest());
     }
 
+    @Test
+    void whenUnknownRecipient_thenReturns400() throws Exception {
+        String requestBody = new String(Files.readAllBytes(acceptanceUnknownRecipient.getFile().toPath()));
+        mockMvc.perform(post("/fhir/Patient/$nhais.acceptance").contentType("application/json").content(requestBody))
+            .andExpect(status().isBadRequest());
```

Comment:
check the response content also to ensure the error is actually caused by an unknown recipient and not some other reason

Reply:
done